index.src.html

```diff
@@ -4256,6 +4309,18 @@
     to specify an offset and stride for each plane in the destination
     {{BufferSource}}. If unspecified, the planes will be tightly packed. It is
     invalid to specify planes that overlap.
+: <dfn dict-member for=VideoFrameCopyToOptions>format</dfn>
+:: A {{VideoPixelFormat}} for the pixel data in the destination
+    {{BufferSource}}. Potential values are:
+    {{RGBA}}, {{RGBX}}, {{BGRA}}, {{BGRX}} and values of {{VideoFrame/format}}.
+    If unspecified or `null`, the {{VideoFrame/format}} is used.
+: <dfn dict-member for=VideoFrameCopyToOptions>colorSpace</dfn>
+:: A {{PredefinedColorSpace}} that <em class="rfc2119">SHALL</em> be used as
+    a target color space for the pixel data in the destination
+    {{BufferSource}}, but only if {{VideoFrameCopyToOptions/format}} is one of
+    {{RGBA}}, {{RGBX}}, {{BGRA}}, {{BGRX}}, otherwise it is ignored.
```

Comment:
It's also necessary that format differs from the frame format, otherwise the result will be in the frame's color space.

We should probably enforce that the color space is only set in the conversion case, because some day we might want to add color conversion to the other paths as well.

Reply:
Good point. I rewrote it so if `format` is explicitly set it's always a conversion case.